spec/views/idv/how_to_verify/show.html.erb_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'idv/how_to_verify/show.html.erb' do
+  selection = Idv::HowToVerifyForm::IPP
+  let(:idv_how_to_verify_form) do
+    Idv::HowToVerifyForm.new(selection: selection)
+  end
+
+  before do
+    allow(view).to receive(:user_signing_up?).and_return(false)
+
+    assign :idv_how_to_verify_form, idv_how_to_verify_form
+    render
+  end
+
+  context 'renders the show template with' do
+    it 'a title and info text' do
```

Comment:
nit - can we change this to something like `it 'has expected title and info text'` so it reads more naturally?

Reply:
Here is how it reads when you run the specs

<img width="398" alt="Screenshot 2024-03-27 at 4 38 19 PM" src="https://github.com/18F/identity-idp/assets/125507397/ab86d409-84aa-4b7e-9aea-be7d85ad2282">

Your change would result in... renders the show template with has expected title and info and text. I initially had expected in each but thought about it being implicitly implied because there was a test for it.  I am willing to change it but wondering with this extra information if you'd still like to see that change.

Reply:
oh lol yes you're right, ignore my comments about updating language